src/main/java/com/rultor/agents/github/CommentsTag.java

```diff
@@ -132,6 +165,84 @@ public CommentsTag(final Github ghub) {
     }
 
     /**
+     * Valid version numbers:
```

Comment:
@JimDeanSpivey I would put this entire version validation thing in another object...